integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/SequenceResourceTestCase.java

```diff
@@ -0,0 +1,87 @@
+package org.wso2.micro.integrator.api;
```

Comment:
Lets add License headers

Reply:
Check other places as well

Reply:
Ah sorry, In my IDE view comment content had been hidden. Will add these